### ISSUE TITLE
Can't build on linux

### ISSUE THREAD

**Issue by Marc34** [washingtondc-emu/washingtondc#16]
hi

I try to build washingtondc on ubuntu 17.04, but I get this message : 

/home/marc/Téléchargements/washingtondc-master/src/io/io_thread.c: In function ‘io_main’:
/home/marc/Téléchargements/washingtondc-master/src/io/io_thread.c:109:25: error: ‘EVLOOP_NO_EXIT_ON_EMPTY’ undeclared (first use in this function)
     int const evflags = EVLOOP_NO_EXIT_ON_EMPTY;
                         ^~~~~~~~~~~~~~~~~~~~~~~
/home/marc/Téléchargements/washingtondc-master/src/io/io_thread.c:109:25: note: each undeclared identifier is reported only once for each function it appears in
CMakeFiles/sh4.dir/build.make:1262 : la recette pour la cible « CMakeFiles/sh4.dir/src/io/io_thread.c.o » a échouée
make[2]: *** [CMakeFiles/sh4.dir/src/io/io_thread.c.o] Erreur 1
CMakeFiles/Makefile2:141 : la recette pour la cible « CMakeFiles/sh4.dir/all » a échouée
make[1]: *** [CMakeFiles/sh4.dir/all] Erreur 2
Makefile:94 : la recette pour la cible « all » a échouée
make: *** [all] Erreur 2

Thank for your help.

**Comment by snickerbockers** [comment]
I've seen that error before, it's because your Linux distro isn't running
the latest version of libevent.  It's caused by a change I made a couple
weeks ago that uses a new feature that's only available in very recent
releases of libevent.

I have a Debian machine that was getting the same build failure and I fixed
it by updating to the latest version (2.1.8-stable on the libevent website:
http://libevent.org/ ).

Eventually I'm going find a way to work around not having this feature.  I
use it because it saves a lot of CPU time, but at the same time I want to
avoid using things that are so new that they aren't in package repos yet.
